backend/kuberay/README.md

```diff
@@ -232,14 +296,23 @@ with 4 Nodes each with 8 NVIDIA-A100-SXM4-80GB GPUs, 64 CPU cores, and 1TB memor
 
 !!! note
 
-    Notice that the only variant with a **full-worker** custom resource 
-    is the one with 8 GPUs. Some actuators, like SFTTrainer, use this 
+    Notice that the only variant with a **full-worker** custom resource
+    is the one with 8 GPUs. Some actuators, like SFTTrainer, use this
     custom resource for measurements that involve reserving an entire GPU node.
 
 We provide [an example set of values](vanilla-ray.yaml) for deploying a
 RayCluster via KubeRay. To deploy it, simply run:
 
     helm upgrade --install ado-ray kuberay/ray-cluster --version 1.1.0 --values backend/kuberay/vanilla-ray.yaml
 
-Feel free to customize it to suit your cluster, such as uncommenting GPU-enabled
-workers.
+In the case the ado operation to be executed requires creating Kubernetes
+resources, the RayCluster to be deployed must be associated with a properly
+configured ServiceAccount like the one described [above](#configuring-a-kubernetes-serviceaccount-for-the-raycluster).
+We provide an [alternative set of values](vanilla-ray-service-account.yaml)
+where the ServiceAccount `ray-deployer` is set for head and worker node.
+To deploy it, simply run:
+
+    helm upgrade --install ado-ray kuberay/ray-cluster --version 1.1.0 --values backend/kuberay/vanilla-ray-service-account.yaml
+
+Feel free to customize the example files provided to suit your cluster,
+such as uncommenting GPU-enabled workers.
```

Comment:
Rather than have a separate file (which we'll have to maintain) we can just add an admonition before the `helm upgrade` command of the original example that tells people to add

```
--set head.serviceAccountName=ray-deployer --set worker.serviceAccountName=ray-deployer
```